examples/pytorch/jknet/README.md

```diff
@@ -0,0 +1,83 @@
+# DGL Implementation of JKNet
+
+This DGL example implements the GNN model proposed in the paper [Representation Learning on Graphs with Jumping Knowledge Networks](https://arxiv.org/abs/1806.03536).
+
+Contributor: [xnuohz](https://github.com/xnuohz)
+
```

Comment:
Did the authors release the code? If so, add a link to their repo.

Reply:
I am not sure whether [this](https://github.com/ShinKyuY/Representation_Learning_on_Graphs_with_Jumping_Knowledge_Networks) is their code.

Reply:
There is a code link in the homepage of the first author: https://people.csail.mit.edu/keyulux/. He did not release a complete example, but we can still check for the model implementations.